docs/angular/navigation.md

```diff
@@ -177,6 +177,54 @@ We're excluding some additional content and only including the necessary parts.
 
 Here, we have a typical Angular Module setup, along with a RouterModule import, but we're now using `forChild` and declaring the component in that setup. With this setup, when we run our build, we will produce separate chunks for both the app component, the login component, and the detail component.
 
+## Standalone Components
+
+:::caution Experimental API
+
+Standalone components is an experimental API introduced in Angular 14.x and available in Ionic 6.3 and later.
```

Comment:
Can we have this also say something along the lines of "This feature may change before it is stable"